packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardMedium.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Link, Typography } from '@mui/material'
+import React from 'react'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../../utils/functions/getEndpoint'
+import { ReleaseCardProps } from './ReleaseCard'
+import { ReleaseCardMediumConfig, ReleaseCardStat } from './ReleaseCardTypes'
+import { formatReleaseCardData } from './ReleaseCardUtils'
+
+const StatGrid: React.FunctionComponent<ReleaseCardStat> = ({
+  value,
+  label,
+}: ReleaseCardStat) => {
+  return (
+    <>
+      <Box gridColumn="span 1" gridRow="span 1" justifySelf="flex-end">
+        <Typography variant="body1" lineHeight="20px">
+          {value}
+        </Typography>
+      </Box>
+      <Box gridColumn="span 1" gridRow="span 1" justifySelf="flex-start">
+        <Typography variant="body1" lineHeight="20px" color="grey.700">
+          {label}
+        </Typography>
+      </Box>
+    </>
+  )
+}
+
+export type ReleaseCardMediumProps = Omit<
+  ReleaseCardProps,
+  'releaseCardConfig'
+> & { releaseCardConfig: ReleaseCardMediumConfig }
+
+export const ReleaseCardMedium: React.FunctionComponent<
+  ReleaseCardMediumProps
+> = ({
+  data,
+  schema,
+  includePortalCardClass = true,
+  releaseCardConfig,
+}: ReleaseCardMediumProps) => {
+  const { releaseVersion, releaseEntity, releaseDate, stats } =
+    formatReleaseCardData(schema, data, releaseCardConfig.statsConfig)
+
+  if (releaseVersion === null) return <></>
+
+  const nCols = 2
+  const SPAN_ALL_COLS = `span ${nCols}`
+  const nRows = stats.length + 4
+
+  return (
+    <Box
+      className={`ReleaseCard ${
+        includePortalCardClass ? 'SRC-portalCard' : ''
+      }`}
+      display="grid"
+      gridTemplateColumns={`repeat(${nCols}, 1fr)`} // columns are controlled by this component, not the parent grid
+      columnGap="10px"
+      gridTemplateRows="subgrid" // rows are controlled by parent grid, so text will be aligned across cards
```

Comment:
Use a grid for columns in each Card, but use the grid of the list of Cards to determine each Card's rows in a subgrid. This ensures that row heights are aligned across Cards, so even if text wraps to the next row in one Card.

Reply:
Can't wait to use this more and more in these portals components!

Based on my inspection, this looks ok for users who use a browser that does not support subgrid (actually imperceptible in the demo because all cards have same # of stats and each sub-row has the same height).